omod/src/main/java/org/openmrs/module/kenyaemr/web/controller/KenyaemrCoreRestController.java

```diff
@@ -2854,6 +2855,23 @@ public List<org.openmrs.module.webservices.rest.SimpleObject> search(@RequestPar
 
     }
 
+    /**
+     * Generates KenyaEMR related metrics
+     * @param request
+     * @return
+     */
+    @RequestMapping(method = RequestMethod.GET, value = "/getemrmetrics")
+    @ResponseBody
+    public Object getKenyaEMRDetails(HttpServletRequest request) {
+
+        return "{\n" +
+                "    \"EmrName\":\"KenyaEMR\",\n" +
+                "    \"EmrVersion\":\"" + DwapiMetricsUtil.getKenyaemrVersion() + "\",\n" +
+                "    \"LastLoginDate\":\"" + DwapiMetricsUtil.getLastLogin() + "\",\n" +
+                "    \"LastMoH731RunDate\":\"" + DwapiMetricsUtil.getDateofLastMOH731() + "\"" +
+                "}";
```

Comment:
What do you think if we use the` SimpleObject so = new SimpleObject()` here? could be cleaner.

Reply:
Makes sense. I just lifted the previous implementation. I will address this.